examples/rococo/assetHub/foreignAssetTransfers/reserve/assetHubWETHToRhala.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetTransferApi, constructApiPromise } from '../../../../../src';
+import { TxResult } from '../../../../../src/types';
+import { GREEN, PURPLE, RESET } from '../../../../colors';
+
+/**
+ * In this example we are creating a `polkadotXcm` pallet `transferAssets` call to send WETH (foreign asset with location `{"parents":"2","interior":{"X2":[{"GlobalConsensus":{"Ethereum":{"chainId":"11155111"}}},{"AccountKey20":{"network":null,"key":"0xfff9976782d46cc05630d1f6ebab18b2324d6b14"}}]}}`)
+ * from a Rococo Asset Hub (System Parachain) account
+ * to a Rhala Testnet (ParaChain) account, where the `xcmVersion` is set to 4, and there is no
```

Comment:
This will not work (make sure to actually validate these on Rococo).

`transferAssets` uses the asset ID to determine its reserve => in this case it will try to use the Ethereum network as the reserve and try to route the transfer through there and will fail because Ethereum is unreachable/across the bridge.

In this case you want AssetHub to act as a LocalReserve -> use `transferAssetsUsingTypeAndThen` just like you did in `assetHubMUSEToRhala.ts`.

Make sure to actually validate on Rococo all of the calls generated by these APIs!

Reply:
Hey Adrian, thanks for adding this additional context! When testing submitting this tx in particular, it appeared to succeed from the perspective of AH so the assumption I made was based on that. I'll be sure to update this example to use `LocalReserve` as you mentioned.

Reply:
Make sure to check for the intended side-effects as well (transfer reaches and correctly handled on receiving side). It can happen that something works on the sending side but will not work on the recipient side.